src/transformers/trainer.py

```diff
@@ -300,13 +308,13 @@ def get_test_dataloader(self, test_dataset: Dataset) -> DataLoader:
         """
         Returns the test :class:`~torch.utils.data.DataLoader`.
 
-        Will use no sampler if :obj:`test_dataset` is a :obj:`torch.utils.data.IterableDataset`, a sequential
+        Will use no sampler if :obj:`test_dataset` does not implement :obj:`__len__`, a sequential
         sampler (adapted to distributed training if necessary) otherwise.
 
         Subclass and override this method if you want to inject some custom behavior.
 
         Args:
-            eval_dataset (:obj:`torch.utils.data.dataset.Dataset`, `optional`):
+            test_dataset (:obj:`torch.utils.data.dataset.Dataset`, `optional`):
```

Comment:
Why the name change? The variable in the signature is still `eval_dataset`, which is coherent with the signature name `get_eval_dataloader`

Reply:
There are 2 functions:
- `get_eval_dataloader` has argument `eval_dataset` (L280-L305)
- `get_test_dataloader` has argument `test_dataset` (L307-L329)

The name did not change, but the diff makes it look like it changed indeed.